index.html

```diff
@@ -36,45 +38,308 @@ <h2 class="greeting">
+                                  </a>
+                                </li>
+                                <li>
+                                  <a href="linkdin.com">
+                                    <img class="icons" src="./images/Icon/LinkedIn.png" alt="LinkedIn">
+                                  </a>
+                                </li>
+                                <li >  
+                                  <a href="">
+                                  <img  class="icons" src="./images/Icon/Disabled.png" alt="disabled">
+                                </a>
+                              </li>
+                                <li >
+                                  <a href="#">
+                                    <img class="icons" src="./images/Icon/Github.png" alt="github">
+                                  </a>
+                                </li>
+                                <li >
+                                  <a href="#">
+                                    <img class="icons" src="./images/Icon/Valid.png" alt="valid">
+                                  </a>
+                                </li>
+                          </ul>
+                    </div>
+                   
+                        <!-- ---------------------------Languages--------------------- -->
+                          <div class="grid separate">
+                            <div class="soft-skill">
+                              <h5 class="language">Language</h5>
+                              <img src="./images/Icon/DisabledDownm.png" alt="disabled cheveron">
+                            </div>
+                          </div>
+                      
+                        <div class="grid-container">
+                            <div class="web-lang">
+                              <img src="./images/Icon/Ellipse 1.png" alt="Javascript">
+                              <h5>Javascript</h5>
+                            </div>
+
+                            <div class="web-lang">
+                                <img src="./images/Icon/Ellipse 2.png" alt="Html">
+                                <h5>HTML</h5>
+                            </div>
+                        
+                        <div class="web-lang">
+                            <img src="./images/Icon/Ellipse 3.png" alt="CSS">
+                            <h5>CSS</h5>
+                          </div>
+                        </div>
+
+                        <div class="grid separate">
+                              <div class="soft-skill">
+                                <h5 class="language">FrameWorks</h5>
+                                <img src="./images/Icon/CodingLine.png" alt="code cheveron">
+                              </div>
+
+                              <div class="soft-skill">
+                                  <h5 class="language">Skills</h5>
+                                  <img src="./images/Icon/CodingLine.png" alt="code cheveron">
```

Comment:
- [ ] As requested by the previous code review , it could be better to make sure that the style of these Items stick to the Figma design 👍. I really appreciate the effort you put in it but make sure you improve the border styling 🤙 

![Capture](https://user-images.githubusercontent.com/80458199/229095570-64237624-7b1f-4a10-ba7c-3859c3ddf9b4.PNG)